cnd/cnd.lexer/src/org/netbeans/cnd/api/lexer/CndLexerUtilities.java

```diff
@@ -448,12 +448,16 @@ public static Filter<?> getFilter(Language<?> language, /*can be null*/CndLangua
                     case C89:
                     case C99:
                     case C11:
+                    case C17:
+                    case C23:
                         return CndLexerUtilities.getHeaderC11Filter();
                     case CPP98:
                         return CndLexerUtilities.getHeaderCppFilter();
                     case CPP11:
                     case CPP14:
                     case CPP17:
+                    case CPP20:
+                    case CPP23:
```

Comment:
I'd add a comment here like so:

```java
// TODO-CND: Improve CndLexerUtilities.getHeaderCpp1Filter() to handle CPP20 and CPP23
```

Reply:
Thanks for taking the time to review this!

I thought the CND lexer was obsolete after the switch to LSP.  If that's not the case, I can take a crack at updating the tokens too based on [cpp reference](https://en.cppreference.com/w/cpp/keyword).